hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSource.java

```diff
@@ -840,7 +844,7 @@ public void uncaughtException(final Thread t, final Throwable e) {
 
     // If this is a recovered queue, the queue is already full and the first log
     // normally has a position (unless the RS failed between 2 logs)
-    private long getRecoveredQueueStartPos(long startPosition) {
+    public long getRecoveredQueueStartPos(long startPosition) {
```

Comment:
Is this change needed? I don't see any changes to a caller of this method in this patch

Reply:
No, I will revert it.